### PR TITLE
[BACKLOG-36843]-Accessibility-Browse Files Perspective-Files List

### PR DESCRIPTION
**Pull request by phalguni-a**
@pentaho/wcag, please review

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 14m 6s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl user-console
```
---
:ok_hand: All tests passed!
 
Tests run: **12**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5250/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: user-console/src/main/resources/org/pentaho/mantle/public/browser/js/browser.templates.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ define([
           "<div class='body' role='tree' aria-labelledby='foldersHeader'></div>" +
           "</div>" +
           "<div id='fileBrowserFiles' class='span4 well fileBrowserColumn'>" +
-          "<div class='body'></div>" +
+          "<select class='body' aria-labelledby='filesHeader' multiple size='0' id='MyID'></select>" +
```
Comment (dcleao) [inline user-console/src/main/resources/org/pentaho/mantle/public/browser/js/browser.templates.js:34]: Could you give a more meaningful identifier? Or, is it forgotten and not needed?

**Review by dcleao — CHANGES_REQUESTED** [review]

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 13m 56s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl user-console
```
---
:ok_hand: All tests passed!
 
Tests run: **12**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5250/4/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by dcleao — APPROVED** [review]

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 13m 20s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl user-console
```
---
:ok_hand: All tests passed!
 
Tests run: **12**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5250/5/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by ddiroma — APPROVED** [review]

**Pull request merged**